contrib/rv2tle.py

```diff
@@ -73,14 +73,14 @@ def rvel(r, v):
 
     vk = 1.0 / XKE * vv2  # smult
     h = np.cross(rr2, vk)  # cross
-    pl = np.dot(h, h)
+    pl = h @ h
     vz = np.array([0.0, 0.0, 1.0])
     n = np.cross(vz, h)
     n = unitv(n)
     rk = np.linalg.norm(rr2)
```

Comment:
In other lines, the `np.linalg.norm` is replaced by `np.sqrt(vec @ vec)`. The `norm` function is more explicit, but I wonder which one is faster 🤔

Reply:
Another way of doing so is using: `(vec @ vec) ** 0.5`, which is applied in other lines  too.

Reply:
Interestingly, the last approach is faster!

```
In [7]: np.sqrt(u @ u)
Out[7]: 0.9193857259765762

In [8]: norm(u)
Out[8]: 0.9193857259765762

In [9]: (u @ u) ** .5
Out[9]: 0.9193857259765762

In [10]: %timeit np.sqrt(u @ u)
1.32 µs ± 5.52 ns per loop (mean ± std. dev. of 7 runs, 1000000 loops each)

In [11]: %timeit norm(u)
2.16 µs ± 21.7 ns per loop (mean ± std. dev. of 7 runs, 100000 loops each)

In [12]: %timeit (u @ u) ** .5
708 ns ± 7.26 ns per loop (mean ± std. dev. of 7 runs, 1000000 loops each)

In [13]: def unorm(u):
    ...:     return (u @ u) ** .5
    ...: 

In [14]: %timeit unorm(u)
753 ns ± 16.6 ns per loop (mean ± std. dev. of 7 runs, 1000000 loops each)
```

With numba though, the differences are not so big:

```
In [15]: from numba import njit

In [16]: @njit
    ...: def unorm1(u):
    ...:     return np.sqrt(u @ u)
    ...: 

In [17]: @njit
    ...: def unorm2(u):
    ...:     return norm(u)
    ...: 
    ...: 

In [18]: @njit
    ...: def unorm3(u):
    ...:     return (u @ u) ** .5
    ...: 

In [19]: unorm1(u)
Out[19]: 0.9193857259765762

In [20]: unorm2(u)
Out[20]: 0.9193857259765762

In [21]: unorm3(u)
Out[21]: 0.9193857259765762

In [22]: %timeit unorm1(u)
172 ns ± 1.05 ns per loop (mean ± std. dev. of 7 runs, 10000000 loops each)

In [23]: %timeit unorm2(u)
299 ns ± 9.15 ns per loop (mean ± std. dev. of 7 runs, 1000000 loops each)

In [24]: %timeit unorm3(u)
203 ns ± 2.16 ns per loop (mean ± std. dev. of 7 runs, 10000000 loops each)
```

But I agree we should be consistent:

1. Add a fast `norm` implementation in `poliastro.core.util` (or even the new upcoming `poliastro._math`)
2. Use it everywhere

Reply:
Opened #1433 to continue that discussion

Reply:
Wow, thanks for the deep review! Yeah, I was thinking about the recently proposed `poliastro._math` module.